addons/resource.language.en_gb/resources/strings.po

```diff
@@ -16138,7 +16138,10 @@ msgctxt "#24162"
 msgid "Use Kodi virtual keyboard"
 msgstr ""
 
-#empty string with id 24163
+#: system/settings/darwin_tvos.xml
+msgctxt "#24163"
```

Comment:
not quite sure if the existing placeholder is there for a reason. Do you know if it's safe to use it?

Reply:
@gade01 or @anxdpanic could answer this but I think the translation system can handle it.